images/linux/scripts/tests/Android.Tests.ps1

```diff
@@ -0,0 +1,54 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1" -DisableNameChecking
+Import-Module "$PSScriptRoot/../software-report/SoftwareReport.Android.psm1" -DisableNameChecking
```

Comment:
`helpers/Common.Helpers.psm1` should be already imported. We don't need to duplicate it.
`software-report/SoftwareReport.Android.psm1` - I am not sure that it is convenient to import SoftwareReport.Android here. If some function are reusable between tests and software reports, I suggest to put them to the `helpers/Android.Helpers.psm1`

Reply:
Done, thank you!